### PR TITLE
Add dashboard ew2020

### PR DESCRIPTION
**Pull request by flsobral**
Adds https://github.com/TotalCross/dashboard-sample to this repo

### CODE REVIEW

**Comment by otavio** [comment]
It'd be nice to squash the commit so it provides a linear and clean commit history.

File: dashboard-ew2020/src/main/java/com/totalcross/sample/dashboard/RunDashboardSampleApplication.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.totalcross.sample.dashboard;
+
+import totalcross.TotalCrossApplication;
+
+public class RunDashboardSampleApplication {
+    public static void main(String[] args) {
+        // TotalCrossApplication.run(DashboardSample.class, "/scr", "1024x600x24", "/r",
+        // "5443444B3018A93884C5D622");
+        TotalCrossApplication.run(DashboardSample.class, "/scr", "848x480x24", "/r", "5443444B3018A93884C5D622");
```
Comment (VaneskaSousa) [inline dashboard-ew2020/src/main/java/com/totalcross/sample/dashboard/RunDashboardSampleApplication.java:9]: You need to delete the totalcross key from the parameters and comments

**Review by VaneskaSousa — CHANGES_REQUESTED** [review]
Thank you for correcting English in the general README.

My only observation is that the example is using version 6.1.0 and still uses the TotalCross key.

File: dashboard-ew2020/src/main/java/com/totalcross/sample/dashboard/RunDashboardSampleApplication.java (hunk 1 above)
Comment (flsobral) [reply dashboard-ew2020/src/main/java/com/totalcross/sample/dashboard/RunDashboardSampleApplication.java]: done

**Pull request merged**